src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -45,6 +45,9 @@
     <PackageReference Include="MessagePack" Version="1.9.11" />
     <PackageReference Include="MessagePackAnalyzer" Version="1.7.3.7" />
     <PackageReference Include="Microsoft.Extensions.Configuration" Version="$(ExtensionsVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="$(ExtensionsVersion)" />
```

Comment:
We don't really need this line since it will come in transitively (for example from the `Microsoft.Extensions.Configuration` above.

```suggestion
```